finesse/gui/interferometer_monitor.py

```diff
@@ -127,10 +88,22 @@ def _display_props(self) -> None:
             lineedit = self._get_prop_lineedit(prop)
             lineedit.setText(prop.val_str())
 
-    def poll_server(self) -> None:
+    def get_data_table(self, data: list[EM27Property]):
+        """Receive the data table from the server."""
+        self._data_table = data
+
+    def begin_polling(self) -> None:
+        """Initiate polling the server."""
+        self._poll_light._timer.start(2000)
+
+    def end_polling(self) -> None:
+        """Terminate polling the server."""
+        self._poll_light._timer.stop()
+
+    def _poll_server(self) -> None:
         """Polls the server to obtain the latest values."""
         self._poll_light._flash()
```

Comment:
I missed this one - if it is going to be used outside of `poll_light`, `flash` also need to be public.